### PR TITLE
Better rotating strategy for auto scaling

### PR DESCRIPTION
**Pull request by krisdante**

### CODE REVIEW

File: roles/cs.aws-autoscaling/tasks/main.yml [hunk 1]
```diff
@@ -26,19 +26,53 @@
...
+
+  - name: Reduce AutoScaling Group - remove old instances
+    ec2_asg:
+      state: present
+      name: "{{ autoscaling_asg_name }}"
+      region: "{{ aws_region }}"
+      min_size: "{{ autoscaling_asg_min_size }}"
+      max_size: "{{ autoscaling_asg_max_size }}"
+      desired_capacity: "{{ autoscaling_asg_desired_capacity }}"
+      termination_policies: [ 'OldestLaunchConfiguration', 'NewestInstance' ]
```
Comment (Szpadel) [inline roles/cs.aws-autoscaling/tasks/main.yml:70]: Shouldn't it have here `OldestInstance`? and should it be here `autoscaling_asg_termination_policies` + OldestLaunchConfiguration?

Comment (krisdante) [reply roles/cs.aws-autoscaling/tasks/main.yml:70]: No, first we remove those with OldestLaunchConfiguration. Fron those that left we remove NewestOnce (we always did it this way)

Comment (krisdante) [reply roles/cs.aws-autoscaling/tasks/main.yml:70]: Doesn;t really matter as the Cron node will die anyway

Comment (Szpadel) [reply roles/cs.aws-autoscaling/tasks/main.yml:70]: but we are loosing option to configure that with `autoscaling_asg_termination_policies` and that option is now useless, therefore we should either remove that variable, or use it here

**Pull request merged**